sphinx/ext/autosectionlabel.py

```diff
@@ -60,6 +79,7 @@ def register_sections_as_label(app: Sphinx, document: Node) -> None:
 def setup(app: Sphinx) -> ExtensionMetadata:
     app.add_config_value('autosectionlabel_prefix_document', False, 'env')
     app.add_config_value('autosectionlabel_maxdepth', None, 'env')
+    app.add_config_value('autosectionlabel_full_reference', False, 'env')
```

Comment:
```suggestion
    app.add_config_value('autosectionlabel_full_reference', False, 'env', bool)
```

Reply:
Does this mean that bool needs to be added to this config variable too? i.e.
 app.add_config_value('autosectionlabel_prefix_document', False, 'env', bool)

Reply:
Yes! That's how GitHub suggestions works (namely, you see the diff of my suggestion with your existing code). If you're wondering what `bool` means in this case, this simply means that we expect `autosectionlabel_prefix_document` to be of `bool` type.